### PR TITLE
changes to index

### PR DESCRIPTION
**Pull request by ramon349**
Changed exec. Also included links to link state 2017 and referenced  the next HACKWRU instead ofHACKCWRU 2016.

### CODE REVIEW

File: app/views/acm/index.haml [hunk 1]
```diff
@@ -66,13 +59,11 @@
   Every spring ACM holds our HackCWRU hackathon to bring students from
   schools near and far to Case Western Reserve University for food,
   sleeplessness, caffeine, coding, hardware, and (most importantly) learning.
-  HackCWRU 2016 will take place from February 12th to February 14th. You can
-  follow these helpful links to get more information:
+  HackCWRU 2018 is currently in the planning stages.
```
Comment (brenns10) [inline app/views/acm/index.haml]: I believe we're working on HackCWRU 2017?

Comment (ajm188) [inline app/views/acm/index.haml]: HackCWRU 2017 definitely already happened this year 😛

Comment (ramon349) [inline app/views/acm/index.haml]: So if this is okay. I'll go ahead and merge right?

File: app/views/acm/index.haml [hunk 2]
```diff
@@ -3,27 +3,20 @@
   %h2 Contact Us
   %ul.aside-list
     %li
-      %a.key{ href: 'mailto:acv31@case.edu' } Anno van den Akker
+      %a.key{ href: 'mailto:emf65@case.edu' } Elizabeth Fregoso 
       .value President
     %li
-      %a.key{ href: 'mailto:mtb89@case.edu' } Matthew Bentley
+      %a.key{ href: 'mailto:yxh597@case.edu' } Yidi  Huang
```
Comment (ajm188) [inline app/views/acm/index.haml]: looks like you've got two spaces between "Yidi" and "Huang"

File: app/views/acm/index.haml [hunk 3]
```diff
@@ -3,27 +3,20 @@
...
     %li
-      %a.key{ href: 'mailto:mtb89@case.edu' } Matthew Bentley
+      %a.key{ href: 'mailto:yxh597@case.edu' } Yidi  Huang 
       .value VP of ACM
     %li
-      %a.key{ href: 'mailto:mlm232@case.edu' } Morgan McMahon
-      .value VP of IEEE
-    %li
-      %a.key{ href: 'mailto:bgr19@case.edu' } Ben Roytenberg
+      %a.key{ href: 'mailto:sxc868@case.edu' }Sacheth Chandramouli
```
Comment (ajm188) [inline app/views/acm/index.haml]: nit: add a space after the "}"?

**Pull request merged**